backend/.env.example

```
@@ -1,2 +1,11 @@
-SERVER_ADDR=127.0.0.1:9894
\ No newline at end of file
+SERVER_ADDR=127.0.0.1:9894
+
+# JWT Configuration
+JWT_SECRET=your-super-secret-jwt-key-change-in-production
+
+# Refresh Token Configuration
+REFRESH_TOKEN_TTL_DAYS=7
+
+# Account lock duration in minutes (when token theft is detected)
+ACCOUNT_LOCK_DURATION_MINUTES=30
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)